docs/vault/guide.md

```diff
@@ -137,6 +137,17 @@ Now you must send the BTC to the Bitcoin address **generated by the parachain**.
 
 You Vault only starts earning rewards once BTC is locked - and rewards are determined by your share of the total BTC locked in the system. Hence, you can increase your rewards by bringing your own BTC into the system.
 
+### How to avoid sniping while minting?
+- You can go to polkadot.js.org/apps and deactivate accepting new issue requests from Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, false)
+- After 36 hours (normally 24 hours, but blocktimes), all the pending issue request without a BTC transaction will be cancelled
+- No one can mint against your collatreal while issuing is turned false so you can add collateral without a worry.
+- For self-minting, you can batch:
+1. Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, true)
+2. Developer -> Extrinsics -> issue -> requestIssue
+3. Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, false)
+![Screenshot: Creating batchAll call to avoid sniping your collateral](../_assets/img/vault/batchallcall.png)
```

Comment:
is this image missing? Maybe I'm being dense, but don't see it in this PR or on `master` https://github.com/interlay/interlay.github.io/tree/master/docs/_assets/img/vault

Reply:
you correct, added it now https://github.com/interlay/interlay.github.io/pull/214/commits/918faef4712be2faf8b2d880de3f4ccb7ef10669